### DISCUSSION TITLE
Application stuck on "Processing information..."

### DISCUSSION THREAD

**Discussion by HoggScripts** [community/community#101082] [GitHub Education]
### Hi everyone,

Hi everyone,

Github student developer pack wont let me register. After i upload my student ID, I click the "process information"  button and it stays on "process information..." indefinitely. I've tried going back and doing this over several times and it always does the same thing. I've tried deleting cookies and using different browsers. I've attached a screenshot. How can I get this fixed?
<img width="1313" alt="Screenshot 2024-01-21 at 10 26 12 PM" src="https://github.com/community/community/assets/143638417/c181d9d3-065a-4602-854e-bab2ca683edd">

Thank you!

**Comment by Justice52** [comment]
I'm facing same issue.
![git](https://github.com/community/community/assets/108245290/12540156-5650-4246-8b11-e33a2f1db556)

**Comment by erxand** [comment]
I'm facing the same issue
![image](https://github.com/community/community/assets/97632407/debebcbc-88e2-42fa-9877-61054825f18c)

**Comment by Mohamed-Emad126** [comment]
I had the same issue but it was solved once I took the picture from the submission page directly, don't upload edited photos and use the take photo button on the submission page this will work, and submit your application.

**Reply by AgamJain05** [reply]
can you explain it more clearly? How can i fix this issue?

**Reply by Mohamed-Emad126** [reply]
Sure: 
- Open the [submission link](https://education.github.com/discount_requests/application) from any **mobile browser** 
- then choose your email and school or university on the first page as usual.
- finally, **don't upload the image** from your mobile, take it directly, I mean **click on **Take Picture** and take a good photo of your ID then upload it**.

Note that it's not necessary to be Perfect or FHD or 4K even, just the information on the ID looks clear enough to be processed fast, if u do this it may take less than a day till the approval, I did that when I wrote the replay before 3 hrs, now I get the approval.

**Reply by AgamJain05** [reply]
thank you so much!! It worked

**Reply by Mohammad-Ali-Yesar** [reply]
I have tried uploading my picture so many times. Taking the picture instead works wonders - thanks a lot @Mohamed-Emad126
